packages/cardano-services/test/cli.test.ts

```diff
@@ -1667,13 +1667,71 @@ describe('CLI', () => {
               );
               await assertServiceHealthy(apiUrl, ServiceNames.Asset, lastBlock);
 
-              const res = await axios.post(`${apiUrl}/asset/get-asset`, {
+              const res = await axios.post<Asset.AssetInfo>(`${apiUrl}/asset/get-asset`, {
                 assetId: asset.id,
                 extraData: { tokenMetadata: true }
               });
 
               const { tokenMetadata } = fromSerializableObject<Asset.AssetInfo>(res.data);
-              expect(tokenMetadata).toStrictEqual({ name: asset.name });
+              expect(tokenMetadata).toStrictEqual({ assetId: asset.id, name: asset.name });
+            });
+
+            it('exposes a HTTP server with healthy state when using CLI options and reaching /get-assets', async () => {
```

Comment:
```suggestion
            it('exposes a HTTP server with healthy state when using CLI options, and /get-assets returns successfully', async () => {
```

Reply:
done in [da0d689](https://github.com/input-output-hk/cardano-js-sdk/pull/691/commits/da0d6895c9995f98aa783ee29399aae26fe11a70)